pom.xml

```diff
@@ -612,6 +624,19 @@
 
   <dependencyManagement>
     <dependencies>
+      <!-- Scala -->
+      <dependency>
+        <groupId>org.scala-lang</groupId>
+        <artifactId>scala-library</artifactId>
+        <version>${scala.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.scala-lang.modules</groupId>
+        <artifactId>scala-collection-compat_${scala.binary.version}</artifactId>
```

Comment:
Hey, these 2 dependencies affect the hudi-flink module, and the local tests can not run now because flink 1.15.x requires scala 2.12.x, can you fix that ?